wordpress-mobile/WordPress-iOS
# Jetpack Social: Add the remaining social shares view to the post settings screen

See: #20784

## Description

Adds the remaining shares view to the post settings screen.

## Screenshots

| Remaining Shares - Light | Remaining Shares - Dark |
|------|------|
| ![Light remaining](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/8efa33c3-3e40-409f-a64a-aa39475c6986) |  ![Dark remaining](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/c4efd746-9490-41e7-9d07-0efbcf1f7997) |

| No Remaining Shares - Light | No Remaining Shares - Dark |
|------|------|
| ![Light no remaining](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/9cfb3178-1717-49bc-a216-0202bacb7637) | ![Dark no remaining](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/6a8af007-7476-4224-85a7-5835987cefb0) |

## Testing

To test:
- Launch Jetpack and login
- Select a site with a social connection
- Create a blog post or edit one
- On the editor, open the menu (`...`) in the top right
- Tap on `Post Settings`
- **Verify** the remaining shares UI matches: Th1ahHKq53k5JT1PNDMavY-fi-1043%3A15560
- Update `displayWarning` in the view model to default to `true`
- Relaunch and navigate back to the post settings screen
- **Verify** the UI with the warning matches: Th1ahHKq53k5JT1PNDMavY-fi-1043%3A16716

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
